src/AuxDisplay.jsx

```diff
@@ -43,12 +46,12 @@ export default class AuxDisplay extends React.Component {
         }
     }
     play() {
-        if (this.el) {
+        if (this.el && this.el.paused) {
```

Comment:
Curious: What's the issue in calling play on a video that is already playing or pause on a video that is already paused?